hw_3_zd_3.js

```diff
@@ -0,0 +1,24 @@
+function deleteDublicate(arr) {
+    let set = new Set(arr);
+    let result = [];
+
+    for (let i = 0; i < set.size - 1; i++) {
```

Comment:
You have 2 cycles - this is not optimized algorithm

Reply:
Done